packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -351,7 +351,7 @@ contract Initializer_Test is Bridge_Initializer {
         // Ensure that all L1, L2 `Initializable` contracts are accounted for, in addition to
         // OptimismMintableERC20FactoryImpl, OptimismMintableERC20FactoryProxy, OptimismPortal2,
         // DisputeGameFactoryImpl, DisputeGameFactoryProxy, DelayedWETHImpl, DelayedWETHProxy.
-        assertEq(_getNumInitializable() + 3, contracts.length);
+        assertEq(_getNumInitializable() + 1, contracts.length);
```

Comment:
see: https://github.com/ethereum-optimism/optimism/pull/10585/files#r1610005599

we additionally subtract a 2 because we have to consider `SystemConfigInterop´ in addition to `OptimismPortalInterop`